src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -20,6 +23,30 @@
    :padding-left      10
    :border-left-color colors/neutral-40})
 
+(def mention-tag-wrapper
+  {:position           :relative
+   :padding-vertical   0
+   :padding-horizontal 0
+   :margin-vertical    0
+   :margin-horizontal  0
+   :flex-direction     :row
+   :align-items        :center})
+
+(def mention-tag
+  {:background-color    colors/primary-50-opa-10
+   :position            :relative
+   :padding-horizontal  3
+   :border-radius       6
+   :margin-bottom       -3
```

Comment:
I'm not sure about using `margin` to fix style issues. They may not look the same on all devices.

You may have noticed that in the iOS screenshot, the alignment looks perfect, but on Android, it may appear slightly off by `1` or `2px` (though this could be a matter of personal perception).

@rahulpsingh How did you land on `-3` for `margin-bottom`?

Reply:
Yes, as the tag sit in the `rn/touchable-opacity` it's automatically vertically aligned. and the difference between line-height and font size is 7, so a margin bottom of 3 issue with misalignment. About this style not being applied to Android devices, it's a known bug that negative margins are inconsistent(in support) in the Android devices.